packages/api-gateway/src/drivers/http/adapters/admin_panel/handlers.js

```diff
@@ -1,11 +1,27 @@
 async function sayHello(req, reply) {
   const result = await this.adminPanelService.sayHello();
-
   return reply.code(200)
     .header('Content-Type', 'application/json; chartset:utf-8')
     .send({ result });
 }
-
+async function changeUserStatus(req, reply) {
+  // TODO: this is not the place for try/catch
+  try {
+    const result = await this.adminPanelService.changeUserStatus(
+      req.params.user_id,
+      req.body.status,
+      req.body.reason,
+    );
+    return reply.code(200)
+      .header('Content-Type', 'application/json; chartset:utf-8')
+      .send(result);
```

Comment:
I noticed that this endpoint does not return a JSON object, it returns plain text... so I suggest to change the response data structure to be a JSON object.

# Evidence
![Screen Shot 2022-04-10 at 11 46 57](https://user-images.githubusercontent.com/28733681/162635024-777d4537-fc97-4640-93bc-f71837224a25.png)

```suggestion
      .send({ result });
```